data/huion-new-1060-plus.tablet

```diff
@@ -0,0 +1,23 @@
+# HUION
+# New 1060 Plus
+#
+
+[Device]
+Name=Huion New 1060 Plus
+ModelName=
+DeviceMatch=usb:256c:006e:HID 256c:006e Pen;usb:256c:006e:HID 256c:006e Pad
+Class=Bamboo
```

Comment:
This works but I have no idea what the right `Class` is. I can't find docs on this.

Reply:
nothing uses the `Class` anyway, we just have to keep it in case *something* at some point used it and we don't want to break compatibility. We used `Bamboo` for the other Huions, good enough for me.